docs/proposal-platforms.md

```diff
@@ -42,15 +42,15 @@ and partially address
 * Note: it will **not** be possible to specify the platform as an environment
   variable in the same way as we can currently specify remote hosts as an
   environment variable (e.g. `platform = $ROSE_ORIG_HOST`).
-  * Unclear why we need this given the Jinja2 variables can be used?
+  * Unclear why we need this given that Jinja2 variables can be used?
   * Using `$ROSE_ORIG_HOST` relies on Rose specifying extra options to set this
     in `[cylc][environment]` which we should avoid if possible.
 
 ## Example platform configurations
 
 ```ini
 [platforms]
-    [[desktop\d\d|laptop\d\d]]
+    [[desktop\d\d,laptop\d\d]]
```

Comment:
Changed to use 2 separate patterns rather than a single one.
I assume this is already handled by parsec so should just work?
It will still be possible to use a single pattern as before if you prefer (since you can use any regular expression recognised by re.fullmatch).